entity-service-impl/src/main/java/org/hypertrace/entity/service/util/DocStoreConverter.java

```diff
@@ -106,6 +120,61 @@ private static boolean isPartOfAttributeMap(String fieldName) {
     return !EntityServiceConstants.ENTITY_CREATED_TIME.equalsIgnoreCase(fieldName);
   }
 
+//  private static void transform(AttributeValue attributeValue, Filter filter) throws IOException {
+//    if (ATTRIBUTES_LABELS_FIELD_NAME.equals(filter.getFieldName()) && filter.getOp() == Op.EQ) {
+//      transformToEqFilterWithValueListRhs(attributeValue, filter);
+//    } else if (ATTRIBUTES_LABELS_FIELD_NAME.equals(filter.getFieldName()) && filter.getOp() == Op.IN) {
+//      transformToOrFilterChainForStrArray(attributeValue, filter);
+//    } else {
+//      transform(attributeValue, filter, isPartOfAttributeMap(filter.getFieldName()));
+//    }
+//  }
+
+  private static void transformToOrFilterChainForStrArray(AttributeValue attributeValue, Filter filter) {
+    String fieldName = filter.getFieldName() + "." + "valueList" + "." + "values";
```

Comment:
why are we concatenating literal strings?

Reply:
I guess I was more asking why `"." + "valueList" + "." + "values"` vs `".valueList.values"`

Reply:
lol. copy paste?

Reply:
fixed.